### PR TITLE
add more log for diagnose

### PR DESCRIPTION
**Pull request by crazycs520**
Add more logs for diagnose.

Here some log example:

```log
[2023/07/27 16:54:36.207 +08:00] [WARN] [region_request.go:615] ["unable to find stores with given labels"] [region=3021] [leader-invalid=false] [labels="[{\"key\":\"zone\",\"value\":\"z99\"}]"]
[2023/07/27 16:54:36.309 +08:00] [INFO] [region_cache.go:2633] ["change store resolve state"] [store=4] [addr=10.0.1.9:21161] [from=resolved] [to=needCheck] [liveness-state=unreachable]
[2023/07/27 16:54:57.687 +08:00] [WARN] [backoff.go:172] ["regionMiss backoffer.maxSleep 40000ms is exceeded, errors:\nregion_id:3021, region_ver:71, store_type:tikv, peer_addr:, error:epoch_not_match:<>  at 2023-07-27T16:54:56.183288+08:00\nregion_id:3021, region_ver:71, store_type:tikv, peer_addr:, error:epoch_not_match:<>  at 2023-07-27T16:54:56.684244+08:00\nregion_id:3021, region_ver:71, store_type:tikv, peer_addr:, error:epoch_not_match:<>  at 2023-07-27T16:54:57.186257+08:00\ntotal-backoff-times: 87, backoff-detail: regionMiss:87\nlongest sleep type: regionMiss, time: 40010ms"]
[2023/07/27 16:55:01.593 +08:00] [INFO] [region_request.go:1365] ["throwing pseudo region error due to no replica available"] [req-ts=443140822382673921] [req-type=Cop] [region="{ region id: 3021, ver: 71, confVer: 65 }"] [region-is-valid=false] [retry-times=0] [replica-read-type=leader] [replica-selector-state=tryFollower] [stale-read=false] [replica-status="peer: 3023, store: 4, isEpochStale: false, attempts: 0, replica-epoch: 26, store-epoch: 26, store-state: resolved, store-liveness-state: unreachable; peer: 4001, store: 1, isEpochStale: false, attempts: 0, replica-epoch: 1, store-epoch: 1, store-state: resolved, store-liveness-state: unreachable; peer: 4004, store: 2, isEpochStale: false, attempts: 0, replica-epoch: 1, store-epoch: 1, store-state: resolved, store-liveness-state: unreachable"] [total-backoff-ms=18010] [total-backoff-times=43] [total-region-errors=]
```

### CODE REVIEW

**Comment by crazycs520** [comment]
@cfzjywxk @you06 @zyguan PTAL

File: kv/store_vars.go [hunk 1]
```diff
@@ -72,3 +74,17 @@ const (
...
+func (r ReplicaReadType) String() string {
+	switch r {
+	case ReplicaReadLeader:
+		return "leader"
+	case ReplicaReadFollower:
+		return "follower"
+	case ReplicaReadMixed:
+		return "mixed"
+	default:
+		return fmt.Sprintf("unknown-%v", byte(r))
```
Comment (you06) [inline kv/store_vars.go:88]: `ReplicaReadLearner` and `ReplicaReadPreferLeader` are missing.

**Review by you06 — COMMENTED** [review]
rest LGTM

File: kv/store_vars.go (hunk 1 above)
Comment (crazycs520) [reply kv/store_vars.go:92]: done.

**Review by ekexium — APPROVED** [review]

**Review by cfzjywxk — APPROVED** [review]

**Pull request merged**